<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPenawaran</class>
 <widget class="QWidget" name="FormPenawaran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePenawaran">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="noPenawaranLabel">
      <property name="text">
       <string>No Penawaran</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noPenawaranLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="iDBarangLabel">
      <property name="text">
       <string>ID Barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="iDBarangLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel_2">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="statusLineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hargaLabel_2">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="hargaLineEdit_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
